frameworks/mlr3automl/setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+HERE=$(dirname "$0")
```

Comment:
for consistency with other packages:
```
HERE=$(dirname "$0")
VERSION=${1:-"stable"}
REPO=${2:-"https://github.com/a-hanf"}
MLR_REPO=${3:-"https://github.com/mlr-org"}
```
and below, you can then use:
```
SUDO Rscript -e 'remotes::install_github("${MLR_REPO}/mlr3pipelines")'
...
 SUDO Rscript -e 'remotes::install_github("${REPO}/mlr3automl")' 
```
this allows to benchmark forks very easily.

Reply:
Thanks for the detailed feedback :)
I made the changes you requested. Let me know if you need anything else.